config/bindings.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+type bindings []string
+
+// WithBinding configures ingress for an endpoint
+//
+// The requestedBindings argument specifies the type of ingress for the endpoint.
+func WithBindings(requestedBindings ...string) interface {
+	HTTPEndpointOption
+	TLSEndpointOption
+	TCPEndpointOption
+	LabeledTunnelOption
+} {
+	ret := bindings{}
+	for _, binding := range requestedBindings {
+		ret = append(ret, binding)
+	}
+	return ret
+}
+
+func (b bindings) ApplyTLS(cfg *tlsOptions) {
+	cfg.Bindings = []string(b)
+}
+
+func (b bindings) ApplyTCP(cfg *tcpOptions) {
+	cfg.Bindings = []string(b)
+}
+
+func (b bindings) ApplyHTTP(cfg *httpOptions) {
+	cfg.Bindings = []string(b)
+}
+
+func (b bindings) ApplyLabeled(cfg *labeledOptions) {
```

Comment:
do we need this for labeled tunnels? was just following what other BindExtra fields did

Reply:
My understanding is that we don't support bindings for labeled tunnels, so this should be removed, thanks for checking!